mivs/templates/emails/game_accepted.txt

```diff
@@ -1,12 +1,12 @@
 {{ game.studio.primary_contact.first_name }},
 
-Congratulations!  Your game submission ({{ game.title }}) for the 2017 MAGFest Indie Videogame Showcase (MIVS) is one of the 57 that have been accepted into the showcase.
+Congratulations!  Your game submission ({{ game.title }}) for the {{ c.ESCHATON.year }} MAGFest Indie Videogame Showcase (MIVS) is one of the 57 that have been accepted into the showcase.
```

Comment:
OK, I didn't feel like writing a utility that counts all the accepted games so this template still needs updating every year, but at least there's fewer things to update.